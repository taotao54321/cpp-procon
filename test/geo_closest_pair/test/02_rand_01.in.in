50
0.037106 0.000968
0.034658 0.036571
0.026995 0.021750
0.008134 0.018425
0.011330 0.037337
0.045562 0.031134
0.009140 0.025557
0.034104 0.024760
0.027119 0.037405
0.034847 0.020564
0.027678 0.004726
0.041743 0.037334
0.042596 0.018649
0.030327 0.006719
0.000675 0.020491
0.018521 0.004830
0.037363 0.043301
0.030256 0.020932
0.024190 0.009344
0.023093 0.022245
0.025695 0.004049
0.033497 0.016824
0.035572 0.031613
0.031007 0.049306
0.045963 0.032011
0.002490 0.004509
0.020801 0.029690
0.012273 0.043547
0.013197 0.003625
0.020151 0.019187
0.030352 0.025665
0.033331 0.000814
0.048018 0.048487
0.034510 0.032480
0.001876 0.000431
0.011750 0.014913
0.026509 0.042754
0.029749 0.015620
0.014710 0.017236
0.037356 0.004184
0.016783 0.038738
0.041629 0.044030
0.048438 0.010931
0.009913 0.024179
0.049427 0.040042
0.002727 0.029126
0.028195 0.036656
0.029929 0.010397
0.038503 0.000597
0.006212 0.009001
