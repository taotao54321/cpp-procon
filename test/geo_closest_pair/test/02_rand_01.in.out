0.0009643288
